hack/.golint_failures

```diff
@@ -236,10 +236,7 @@ pkg/proxy/util
 pkg/proxy/winkernel
 pkg/proxy/winuserspace
 pkg/quota/v1/evaluator/core
-pkg/registry/admissionregistration/mutatingwebhookconfiguration/storage
 pkg/registry/admissionregistration/rest
```

Comment:
The lint failures in this directory also seems to be fixed.
Can you share the reason not to delete this line?